packages/demo-react-native/App/Config/ReactotronConfig.js

```diff
@@ -24,23 +24,19 @@ if (__DEV__) {
       reconnectionAttempts: 10
     }
   })
+    .useAllIncluded({
```

Comment:
I just tested this and i got a red screen `undefined is not a function (near '...useAllIncluded({...')` 
First stacktrace points exactly to this line `ReactotronConfig.js:27`